Gemfile

```diff
@@ -7,8 +7,11 @@ gem "propshaft"
 # Use pg as the database for Active Record
 gem "sqlite3"
 gem "sqlite-vec"
+gem "neighbor"
```

Comment:
This allows us to run KNN queries, but also formats the embeddings for saving in SQLite